src/client/src/orangehrmAttendancePlugin/components/RecordAttendance.vue

```diff
@@ -308,17 +405,16 @@ export default {
       const timezone = guessTimezone();
       const data = {
         date: this.attendanceRecord.date,
-        time: this.attendanceRecord.time,
+        breakStartTime: this.attendanceRecord.breakStartTime,
+        breakEndTime: this.attendanceRecord.breakEndTime,
         note: this.attendanceRecord.note,
         timezoneOffset:
           this.attendanceRecord.timezone?._offset ?? timezone.offset,
         timezoneName: this.attendanceRecord.timezone?.id ?? timezone.name,
+        breakNote: this.attendanceRecord.breakNote,
       };
       try {
-        if (
-          this.latestAttendanceRecord &&
-          this.latestAttendanceRecord?.attendanceType.id === 'WORK_TIME'
-        ) {
+        /*if (this.latestAttendanceRecord?.attendanceType?.id === 'WORK_TIME') {
```

Comment:
Could we remove the documented code here, as it unnecessarily complicates the function and is not used?